src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.scss

```
@@ -0,0 +1,26 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-tooltip-background-hover: #444;
+  --sbb-tooltip-background-active: #5a5a5a;
+
+  display: inline-block;
+}
+
+.sbb-tooltip-trigger {
+  @include sbb.button-reset;
```

We should inherit the text styles if it's embedded into a text. Maybe it's necessary to also include this text-inherit mixin on host level. Please test the scenario if there is a parent element having a custom font-family, font-weight, font-size, color, line-height, others? and see how the tooltip trigger reacts. Take the custom content story for that.

```suggestion
  @include sbb.button-reset;
  @include sbb.text-inherit;
```